docs/SupportedONNXOps-cpu.md

```diff
@@ -49,7 +49,7 @@ Onnx-mlir currently supports ONNX operations targeting up to opset 16. Limitatio
 | **Cos** |7 | | |
 | **Cosh** |9 | | |
 | **CumSum** |14 | | |
-| **DFT** | |unsupported | |
+| **DFT** | |17 | |
```

Comment:
would hold off on that. This file is for CPU support, so we need the lowering to krnl before stating it is supported.

Reply:
OHH okay let me change that back to unsupported then.

Reply:
Fixed!